tests/ExampleApps/CMakeLists.txt

```diff
@@ -6,121 +6,88 @@ project(RadiumExampleApps)
 # short convenience target from cmake/cleanCpppExtensions.cpp
 add_custom_target(${PROJECT_NAME})
 
-# These variable are normally set by the find_package(Radium) for client applications
-set(RADIUM_ROOT_DIR "${CMAKE_INSTALL_PREFIX}/")
-set(RADIUM_RESOURCES_DIR "${CMAKE_INSTALL_PREFIX}/Resources/Radium")
-set(RADIUM_PLUGINS_DIR "${CMAKE_INSTALL_PREFIX}/Plugins")
-set(IN_RADIUM_BUILD_TREE TRUE)
-
-if(MSVC OR MSVC_IDE OR MINGW)
-    # must find the Qt installation directory. This will be set by RadiumConfig.cmake after install
-    # TODO : find a way to that more efficiently
-    find_qt_dependency(COMPONENTS Core REQUIRED)
-    add_imported_dir(FROM Qt::Core TO QtDlls_location)
+# Test if this cmake project is included and built in the Radium project build tree
+if(${CMAKE_PROJECT_NAME} STREQUAL "radiumproject")
+    # Configure in_project build ... These variable are normally set by the find_package(Radium) for
+    # client applications
+    set(RADIUM_ROOT_DIR "${CMAKE_INSTALL_PREFIX}/")
+    set(RADIUM_RESOURCES_DIR "${CMAKE_INSTALL_PREFIX}/Resources/Radium")
+    set(RADIUM_PLUGINS_DIR "${CMAKE_INSTALL_PREFIX}/Plugins")
+    # This allow to compile the example apps without searching the Radium package when building the
+    # demos in the Radium buildtree
+    set(IN_RADIUM_BUILD_TREE TRUE)
+else()
+    # add a convenience macro for adding a run target
+    macro(add_run_and_debug_targets TARGET)
+        add_custom_target(
+            run_${TARGET}
+            WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+            USES_TERMINAL
+            DEPENDS ${TARGET}
+            COMMAND ./${TARGET}
+        )
+    endmacro()
```

Comment:
I don't get the point here, either move this macro from CleanCppExtensions to RadiumSetupFunction, or remove the use of this macro.

Reply:
I prefer to remove the use of this macro. This kind of target seems useless to me. I think you added them some time ago, so I keep them.